mongodb-app-dependencies/pom.xml

```diff
@@ -2,19 +2,20 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<groupId>org.springframework.cloud.stream.app</groupId>
-	<artifactId>mongodb-app-dependencies</artifactId>
-	<version>1.1.2.BUILD-SNAPSHOT</version>
-	<packaging>pom</packaging>
-	<name>mongodb-app-dependencies</name>
-	<description>Spring Cloud Stream Mongodb App Dependencies</description>
 
 	<parent>
 		<artifactId>spring-cloud-dependencies-parent</artifactId>
 		<groupId>org.springframework.cloud</groupId>
 		<version>1.2.1.RELEASE</version>
 		<relativePath/>
 	</parent>
+	
+	<groupId>org.springframework.cloud.stream.app</groupId>
+	<artifactId>mongodb-app-dependencies</artifactId>
+	<version>1.1.2.BUILD-SNAPSHOT</version>
+	<packaging>pom</packaging>
+	<name>mongodb-app-dependencies</name>
+	<description>Spring Cloud Stream Mongodb App Dependencies</description>
```

Comment:
IMHO `parent`  is really "parent" and he is the Boss here!

Just try to follow with other conventions, e.g. in Spring Boot: https://github.com/spring-projects/spring-boot/blob/master/spring-boot-autoconfigure/pom.xml.
Or Spring Cloud Contract: https://github.com/spring-cloud/spring-cloud-contract/blob/master/spring-cloud-contract-starters/pom.xml